### PR TITLE
SRE-2145 Warnings-ng 11.2.2 modified API

### PR DESCRIPTION
**Pull request by JohnMalmberg**
Warnings-ng 11.2.2 removed the ignoreFailedBuilds parameter from the recordIssues Step.

Doc-only: false

Required-githooks: true

### Before requesting gatekeeper:

* [ ] Two review approvals and any prior change requests have been resolved.
* [ ] Testing is complete and all tests passed or there is a reason documented in the PR why it should be force landed and forced-landing tag is set.
* [ ] `Features:` (or `Test-tag*`) commit pragma was used or there is a reason documented that there are no appropriate tags for this PR.
* [ ] Commit messages follows the guidelines outlined [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments).
* [ ] Any tests skipped by the ticket being addressed have been run and passed in the PR.

### Gatekeeper:

* [ ] You are the appropriate gatekeeper to be landing the patch.
* [ ] The PR has 2 reviews by people familiar with the code, including appropriate owners.
* [ ] Githooks were used. If not, request that user install them and check copyright dates.
* [ ] Checkpatch issues are resolved.  Pay particular attention to ones that will show up on future PRs.
* [ ] All builds have passed.  Check non-required builds for any new compiler warnings.
* [ ] Sufficient testing is done. Check feature pragmas and test tags and that tests skipped for the ticket are run and now pass with the changes.
* [ ] If applicable, the PR has addressed any potential version compatibility issues.
* [ ] Check the target branch.   If it is master branch, should the PR go to a feature branch?  If it is a release branch, does it have merge approval in the JIRA ticket.
* [ ] Extra checks if forced landing is requested
  * [ ] Review comments are sufficiently resolved, particularly by prior reviewers that requested changes.
  * [ ] No new NLT or valgrind warnings.  Check the classic view.
  * [ ] Quick-build or Quick-functional is not used.
* [ ] Fix the commit message upon landing. Check the standard [here](https://daosio.atlassian.net/wiki/spaces/DC/pages/11133911069/Commit+Comments). Edit it to create a single commit. If necessary, ask submitter for a new summary.

### CODE REVIEW

**Comment by JohnMalmberg** [comment]
Gatekeeper, this is the fix needed to get CI working again.   Until it lands, and PRs are rebased against it, all CI jobs will fail.

**Review by ryon-jensen — DISMISSED** [review]

File: Jenkinsfile [hunk 1]
```diff
@@ -782,7 +782,9 @@ pipeline {
                                          valgrind_stash: 'el8-gcc-nlt-memcheck'
                             recordIssues enabledForFailure: true,
                                          failOnError: false,
-                                         ignoreFailedBuilds: true,
+                                         // parameter below removed with the warnings-ng 11.2.2
+                                         // possibly by mistake.
```
Comment (ashleypittman) [inline Jenkinsfile:786]: https://github.com/jenkinsci/warnings-ng-plugin/releases/tag/v11.0.0 states that the change in API was deliberate.

Comment (phender) [reply Jenkinsfile:786]: Seems like we should the change be to just remove the parameter.

Comment (ashleypittman) [reply Jenkinsfile:786]: Yes, I think that would be best in the short term.

**Review by phender — APPROVED** [review]

**Comment by phender** [comment]
Landing based upon the success of the previous build: https://build.hpdd.intel.com/blue/organizations/jenkins/daos-stack%2Fdaos/detail/PR-14111/2.

**Pull request merged**